client/components/button-group/test/index.js

```diff
@@ -1,25 +1,27 @@
+/**
+ * @jest-environment jsdom
+ */
+import '@testing-library/jest-dom';
```

Comment:
This should now be removed because since #63505 we globally import `jest-dom`.

Reply:
Thank you, just did that!